docker/build-container.sh

```diff
@@ -3,12 +3,37 @@
 export UID=$(id -u)
 export GID=$(id -g)
 
-for arg in $@; do
-    if [[ $arg == clean ]]; then
-        docker rm cv-hardware-parts
-	docker rmi ia-bigdata
-    fi;
+while [[ $# -gt 0 ]]; do
+    case $1 in 
```

Comment:
We could have a `--help`  here, then we might not need details in the README (maybe).

But then it's better to only set up variables in the `case` block instead of actually taking actions (leaving actions for later down the script), then the help calls a help function and exits immediately, so there's no effect in running it with the help.

Reply:
I agree. But:

1- I thought it worth it having the script details in the README file, since different types of researchers (especially those who might not be well-versed with command line) may access this repository. It is a minor thing, but having such information in a README file could facilitate their understanding somehow. On the other hand, we need to be careful about providing a lot of information in the home page.

2- I would move this task to another issue (e.g. refactor and clean the code).

Reply:
Works for me! 😄